JLJSONMapping/Category/NSError+JLJSONMapping.m

```diff
@@ -8,24 +8,24 @@
 
 #import "NSError+JLJSONMapping.h"
 
-NSString * const kObjectMappingDomain = @"com.mydoghatestechnology.objectmapping";
-NSString * const kObjectMappingDescriptionKey = @"JLObjectMappingDescriptionKey";
-NSString * const kObjectMappingFailureReasonKey = @"JLObjectMappingDetailedFailureReasonKey";
+NSString * const kFABObjectMappingDomain = @"com.mydoghatestechnology.objectmapping";
```

Comment:
Do we wanna change the string value of this too? That way they'll really be different domains.